--- patch\1.1.0.5\PDF.php.original 
+++ patch\1.1.0.5\PDF.php 
@@ -354,6 +354,14 @@
 		foreach($history as $h)
 			if ($h['id_order_state'] == _PS_OS_SHIPPING_)
 				$shipping_date = $h['date_add'];
+		/* start deliverydays */
+			if (!isset($shipping_date) &&
+				($module = Module::getInstanceByName('deliverydays')) &&
+				($module->active) &&
+				($deliveryDay = $module->getDate($order->id_cart)))
+				$shipping_date = $deliveryDay;
+		/* end deliverydays */
+
 		$pdf->Ln(12);
 		$pdf->SetFillColor(240, 240, 240);
 		$pdf->SetTextColor(0, 0, 0);
